Components/alert.css

```diff
@@ -0,0 +1,71 @@
+body {
```

Comment:
Applying properties to body element directly would've effect on all other pages

Reply:
Okay

Reply:
Done